index.html

```diff
@@ -86,7 +86,7 @@ <h2 class="multi-post">Multi-post stories</h2>
       </button>
     </div>
```

Comment:
- [ ] Kindly add right and left margin/padding to the header/navbar section as in the Figma design.

- [ ]  Kindly adjust the gap between the social media icons on the left and the headline paragraph as in the Figma design.
#### Design
![header-and-headline-desk-figma](https://user-images.githubusercontent.com/44057863/167276396-1334b55f-6091-4000-90de-07fd659ef9ff.png)
#### Impl
![header-and-headline-desk-impl](https://user-images.githubusercontent.com/44057863/167276400-36592bed-98d4-41c9-a522-09544f7f215e.png)